src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS = 'UNDER_CONSTRUCTION';
+
+const cateXer = (state = [], action) => {
```

Comment:
- [ ] Set an `initialState` as it own variable as demonstarted above in the books file.